src/compiler/crystal/tools/init/template/readme.md.ecr

```diff
@@ -12,6 +12,8 @@ dependencies:
   <%= config.name %>:
     github: <%= config.github_name %>/<%= config.name %>
 ```
+
+and run `shards install`.
```

Comment:
This should be a whole sentence (for example `Then run `shards install`).

Reply:
I think this is fine because `Add this to your application's shard.yml:`, the yaml code and this together is one sentence. So the whole installation is swifty and simply described in one sentence.
"Do this and run this" instead of "Do this. Then run this".

Reply:
It's hard to read that as a whole sentence given it's two paragraphs with a code block element in between.